lib/Drush/Psysh/DrushCommand.php

```diff
@@ -118,7 +125,11 @@ protected function execute(InputInterface $input, OutputInterface $output) {
    *   The command aliases.
    */
   protected function buildAliasesFromConfig() {
-    return !empty($this->config['aliases']) ? $this->config['aliases'] : [];
+    $aliases = !empty($this->config['aliases']) ? $this->config['aliases'] : [];
+
+    return array_filter($aliases, function($alias){
```

Comment:
Totally. When I plug my laptop in and have some power i'll carry on with tidying this up.